spec/controllers/api/v1/workflow_contents_controller_spec.rb

```diff
@@ -22,6 +15,14 @@
   let(:primary_content) { workflow.primary_content }
 
   describe "#index" do
+    let!(:acl) do
```

Comment:
again, perhaps extract. If they cause errors else where then i don't think it matters, just easier to modify a change in one spot (method perhaps) rather than multiple times.